app/scripts/controllers/registration.js

```diff
@@ -167,7 +167,7 @@ angular.module('confRegistrationWebApp')
       }), true);
     };
 
-    var findAnswersToSave = function(){
```

Comment:
Mmm I think probably because jshint was complaining. I don't remember exactly. This is defined after its usage. JS will hoist a function declaration but will only hoist the variable part of the function expression and will define the value of it later. (At least that's my understanding, correct me if I'm wrong) Functionally I don't think this matters here as the functions using this function aren't called immediately.